aptible/stack.go

```diff
@@ -62,8 +74,8 @@ func (c *Client) GetStack(id int64) (Stack, error) {
 	return stackToReturn, nil
 }
 
-func (c *Client) UNSUPPORTED_GetStackByName(name string) (Stack, error) {
-	stacks, err := c.UNSUPPORTED_GetStacks()
+func (c *Client) GetStackByName(name string) (Stack, error) {
+	stacks, err := c.GetStacks()
```

Comment:
Line 85 (the return), re-creating the Stack object seems unnecessary. We have the Stack from `GetStacks()` so we can just return it.

Reply:
fair point, doing so now

Reply:
What I mean is instead of this when we find the matching Stack:

```go
return Stack{
  ID:             stack.ID,
  Name:           stack.Name,
  OrganizationID: stack.OrganizationID,
}, nil
```

We can just return the Stack we get back from `GetStacks()`:

```go
return stack, nil
```

Reply:
wow i had a few dupes/useless vars during refactoring from the client pointer junk. updating this as well (3 spots i think)